lib/auth.js

```diff
@@ -486,6 +486,12 @@ function _verifyPasswordHash(identity, type, callback) {
           'Could not verify Identity ' + type + '. Identity is not active.',
           'IdentityInactive'));
       }
+      if(!record.meta['bedrock-authn-password']) {
+        return callback(new BedrockError(
+          'Could not verify Identity ' + type +
+          '. Password-based authentication disabled.',
+          'MethodNotAllowed'));
+      }
```

Comment:
+1 to ensuring `bedrock-authn-password` is set -- I could have sworn we already did this, but I guess not.